six/modules/c++/six.sicd/include/six/sicd/AreaPlaneUtility.h

```diff
@@ -0,0 +1,81 @@
+/* =========================================================================
+ * This file is part of six.sicd-c++
+ * =========================================================================
+ *
+ * (C) Copyright 2004 - 2014, MDA Information Systems LLC
+ *
+ * six.sicd-c++ is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef __SIX_SICD_AREA_PLANE_UTILITY_H__
+#define __SIX_SICD_AREA_PLNAE_UTILITY_H__
```

Comment:
Typo in include guard here (I can say from experience that when these don't quite match, you can pull your hair out for awhile until you stumble on it)